docs/install/README.md

```diff
@@ -79,3 +79,6 @@ versions. This is not recommended for normal users.
 * [upgrading document](../Upgrading.md)
 * [developer guide](../Developer-Guide.md)
 * [runtime documentation](../../src/runtime/README.md)
+
+## Kata Containers 3.0 installation
```

Comment:
Suggestion: IMO it is better to just call it rust-runtime installation guild. Kata 3.0 will maintain both go runtime and rust runtime, where the rust runtime is a feature preview. We do not intend to indicate that the go runtime is deprecated nor the rust runtime is production ready when 3.0 is released.

Reply:
Or, let's call it "Kata Containers 3.0 rust runtime installation"?

Reply:
nvm, saw it is updated. Do you want to add the `3.0` prefix @Tim-0731-Hzt ?

Reply:
ye, we can call it `Kata Containers 3.0 rust runtime installation`

Reply:
fixed